common/proptest-helpers/src/growing_subset.rs

```diff
@@ -118,7 +118,7 @@ where
             if idx >= to_idx {
                 break;
             }
-            self.current_pos += 1;
+            self.current_pos = self.current_pos.saturating_add(1);
```

Comment:
as this is is a `usize` we'll never hit this edge-case, so I think saturating add is as good as any other function here. But since we are in `proptest-helpers`, maybe it'd be simpler to just mark the entire crate as `clippy::allow` instead?

Reply:
sure I'll do that

Reply:
you didn't remove this saturating add